docs/CSBR.md

```diff
@@ -591,6 +1062,52 @@ Prior to using any data source as a Reliable Data Source, the CA SHALL evaluate
 
 Databases maintained by the CA, its owner, or its affiliated companies do not qualify as a Reliable Data Source if the primary purpose of the database is to collect information for the purpose of fulfilling the validation requirements under this [Section 3.2](#32-initial-identity-validation).
 
+### 3.2.8. Denied Lists and Other Legal Block Lists
+
+This section is applicable for EV Certificate Requests.
+
+1. **Verification Requirements**: The CA MUST verify whether the Applicant, the Contract Signer, the Certificate Approver, the Applicant's Jurisdiction of Incorporation, Registration, or Place of Business:
+
+   1. Is identified on any government denied list, list of prohibited persons, or other list that prohibits doing business with such organization or person under the laws of the country of the CA's jurisdiction(s) of operation; or
+   2. Has its Jurisdiction of Incorporation, Registration, or Place of Business in any country with which the laws of the CA's jurisdiction prohibit doing business.
+
+   The CA MUST NOT issue any EV Code Signing Certificate to the Applicant if either the Applicant, the Contract Signer, or Certificate Approver or if the Applicant's Jurisdiction of Incorporation or Registration or Place of Business is on any such list.
+
+2. **Acceptable Methods of Verification**: The CA MUST take reasonable steps to verify with the following lists and regulations:
+
+   1. If the CA has operations in the U.S., the CA MUST take reasonable steps to verify with the following US Government denied lists and regulations:
+
+       1. BIS Denied Persons List - [https://www.bis.doc.gov/index.php/the-denied-persons-list](https://www.bis.doc.gov/index.php/the-denied-persons-list)
+       2. BIS Denied Entities List - [https://www.bis.doc.gov/index.php/policy-guidance/lists-of-parties-of-concern/entity-list](https://www.bis.doc.gov/index.php/policy-guidance/lists-of-parties-of-concern/entity-list)
+       3. US Treasury Department List of Specially Designated Nationals and Blocked Persons - [https://www.treasury.gov/resource-center/sanctions/sdn-list/pages/default.aspx](https://www.treasury.gov/resource-center/sanctions/sdn-list/pages/default.aspx)
+       4. US Government export regulations
+
+   2. If the CA has operations in any other country, the CA MUST take reasonable steps to verify with all equivalent denied lists and export regulations (if any) in such other country.
+
+### 3.2.9 Final Cross-Correlation and Due Diligence
+

```

Comment:
I suggest we here again clarify that this is for EV only:
```suggestion
This section is applicable for EV Certificate Requests.

```

Reply:
It is not just for EV, it is also applicable for OV Certificate Requests. This was done on purpose. Let me know if you need to discuss some more.

Reply:
"due diligence" isn't directly an item in section 11.13 though. 

Considering this may mean the entire section 11.13, that would still only apply to EV Certificates as that section itself call upon the issuance of EV Certificates

Reply:
FWIW, check out the discussion of ballot CSC-7
- https://lists.cabforum.org/pipermail/cscwg-public/2021-January/000257.html

Reply:
As discussed in yesterday's CSCWG Teleconference, in case of any possible misinterpretation or uncertainty, we should add some clarification language to make sure that everyone understands that the requirements for cross-correlation and due diligence apply to both EV and non-EV Code Signing Certificates.

@XolphinMartijn do you want to propose some language around that? I believe it would be non-controversial to add that in the ballot and call it out in the ballot's preamble.

Reply:
@dzacharo I'm good leaving out my proposed suggestion, as mentioned on our call yesterday.

Regarding the remainer of the conversation, I still don't agree with the assessment made there.
Considering we're also moving towards a single type of Code Signing however, I'm also not sure if it's worth diving deeper into that at this point.